### ISSUE TITLE
Opentelemetry Exploration and Implementation

### ISSUE THREAD

**Issue by AshwinKul28** [DiceDB/dice#306]
This issue is to explore and implement opentelemetry package for the dice db and explore about exporters

**Comment by AshwinKul28** [comment]
@JyotinderSingh Please assign it to @gauravsarma1992 and me

**Comment by arpitbbhayani** [comment]
Hello @AshwinKul28,

There has been no activity on this issue for the past 5 days.
It would be awesome if you keep posting updates to this issue so that we know you are actively working on it.

We are really eager to close this issue at the earliest, hence if we continue to see the inactivity, we will have to reassign the issue to someone else. We are doing this to ensure that the project maintains its momentum and others are not blocked on this work.

Just drop a comment with the current status of the work or share any issues you are facing. We can always chip in to help you out.

Thanks again.

**Comment by arpitbbhayani** [comment]
Hello @AshwinKul28,

There has been no activity on this issue for the past 5 days.
It would be awesome if you keep posting updates to this issue so that we know you are actively working on it.

We are really eager to close this issue at the earliest, hence if we continue to see the inactivity, we will have to reassign the issue to someone else. We are doing this to ensure that the project maintains its momentum and others are not blocked on this work.

Just drop a comment with the current status of the work or share any issues you are facing. We can always chip in to help you out.

Thanks again.

**Comment by ayushsatyam146** [comment]
Hi @arpitbbhayani @AshwinKul28 I can try to assist @AshwinKul28 on this issue and get this done if you guys are occupied with something else. Please let me know if I can be of any help.

**Comment by arpitbbhayani** [comment]
Hello @AshwinKul28,

There has been no activity on this issue for the past 5 days.
It would be awesome if you keep posting updates to this issue so that we know you are actively working on it.

We are really eager to close this issue at the earliest, hence if we continue to see the inactivity, we will have to reassign the issue to someone else. We are doing this to ensure that the project maintains its momentum and others are not blocked on this work.

Just drop a comment with the current status of the work or share any issues you are facing. We can always chip in to help you out.

Thanks again.

**Comment by arpitbbhayani** [comment]
Hello @AshwinKul28,

There has been no activity on this issue for the past 5 days.
It would be awesome if you keep posting updates to this issue so that we know you are actively working on it.

We are really eager to close this issue at the earliest, hence if we continue to see the inactivity, we will have to reassign the issue to someone else. We are doing this to ensure that the project maintains its momentum and others are not blocked on this work.

Just drop a comment with the current status of the work or share any issues you are facing. We can always chip in to help you out.

Thanks again.

**Comment by AshwinKul28** [comment]
Closing this as not a priority!
